examples/nextjs-with-typescript/src/pages/_app.tsx

```diff
@@ -4,8 +4,8 @@ import { AppProps } from 'next/app';
 import { ThemeProvider } from '@mui/material/styles';
```

Comment:
The change of folder location is wrong, `/pages` is supposed to be at the same level as the package.json https://nextjs.org/docs/basic-features/pages.

Reply:
@oliviertassinari I mean, Next.js might not be that opiniated but if `/pages` is at the same level as package.json, then there shouldn't be a `/src`. [What's now in `/src`](https://github.com/mui-org/material-ui/tree/master/examples/nextjs-with-typescript/src) should instead be moved into `/components` and `/lib`.

Please take a look at this community post on Dev:
https://dev.to/vadorequest/a-2021-guide-about-structuring-your-next-js-project-in-a-flexible-and-efficient-way-472